go-away/tests/go_marshalling.rs

```diff
@@ -63,6 +74,7 @@ enum FulfilmentType {
 }
 
 #[test]
+#[cfg(feature = "chrono")]
```

Comment:
Would rather not have this entire test disabled if chrono is disabled - could we maybe just add another test for datetimes?

Reply:
will do